script/src/verify/tests/utils.rs

```diff
@@ -135,10 +136,15 @@ impl TransactionScriptsVerifierWithEnv {
         let tmp_dir = TempDir::new().unwrap();
         let db = RocksDB::open_in(&tmp_dir, COLUMNS);
         let store = Arc::new(ChainDB::new(db, Default::default()));
+        let version_1_enabled_at = 10;
```

Comment:
Should we assign different values for `version_1_enabled_at` and `version_2_enabled_at`?